### ISSUE TITLE
Contingency Plan Breakdown

### ISSUE THREAD

**Issue by SanaKhan1997** [holtzmak/Humane-Transport#32]
This issue will focus on the unpredictable incidents, disturbance or accidents occurred while shipment and possible actions or helplines to be considered for such conditions

**Comment by SanaKhan1997** [comment]
@holtzmak : [Contingency Plan Wiki](https://github.com/holtzmak/Humane-Transport/wiki/Contingency-Plan-Breakdown)

**Comment by holtzmak** [comment]
Some slight comments to continue to be consistent with our other breakdowns:

> "Who is responsible for developing a contingency plan for animal transportation?
> ... HAR explains that every transporter and consignee is expected to have their own contingencies plan"
1. Could you specify where this is? Unless it's 138.2, which you state later on...

> Is there a guide to establish a successful contingency plan or to evaluate a pre-existing contingency plan?
> ... The evaluation criteria for a contingency plan as defined by Interpretive Guidance Section 5.0 does is not based on:
2. This sentence is not quite right, did you mean to write something like the following?
**"The evaluation criteria for a contingency plan is defined in Interpretive Guidance Section 5.0, and summarized as follows:"**

> EXAMPLE INFORMATION OF EVENT SPECIFIC CONTINGENCY PLAN/DOCUMENT:

Your examples in page are: *chef's kiss*. Should we update the others to have formatting in page? I think the Animal Transport Record would be hard to do so, due to the images in the template.

 _(These breakdowns are awesome btw, I like having gathered documentation like this that shows we're breaking down the problem we've been given vs. having to explain verbally how and why we got somewhere [like the vlogs])_

**Comment by SanaKhan1997** [comment]
I have made some changes in contingency plan breakdown. Since, I typed this doc somewhere else and copy/pasted here, I noticed there were bunch of errors. 
1. Provided the link of HAR section 138.2 on wiki page which basically says who is responsible for the plan in its first paragraph--[HAR section 138.2 subsection 1](https://laws-lois.justice.gc.ca/eng/regulations/c.r.c.,_c._296/page-16.html#h-1226904)
2. There were errors in evaluation criteria due to copy/paste. The first few lines explains what is not considered when evaluating and later it mentions some suggestions for a good plan.
3. I personally really liked that we have example information from templates for possible forms. I think it would make it easier for anyone who is working on the design of mega-form for our app. If you want I can edit other wikis to have similar example information for each form.

Lastly, thanks for your feedback and pointing out some errors : )
